integrations/adobe-analytics/lib/index.js

```diff
@@ -1082,13 +1082,23 @@ function createStandardVideoMetadata(track, mediaObj) {
 function createCustomVideoMetadataContext(track, options) {
   var contextData = {};
 
+  //Check properties object for `settings.contextValue` mappings to assign custom metadata
   var properties = extractProperties(trample(track.properties()), options);
   each(function(value, key) {
     if (!key || value === undefined || value === null || value === '') {
       return;
     }
     contextData[key] = value;
   }, properties);
+
+  //Check properties object for `settings.contextValue` mappings to assign custom metadata
+  var contextFields = extractProperties(trample(track.context()), options);
```

Comment:
`track.context()` doesn't actually return the context object. It's an alias for the integration options. 😕 

See: https://github.com/segmentio/facade/blob/a7cc955f57e14373bb4c502a3ea54a96d8aef9bc/lib/facade.js#L215-L215

Reply:
I'm assuming they want to pass things like IP address or user agent.

Reply:
@gpsamson yes that is correct. The customer wants to send in nested context fields. That is interesting, since that is what we have been using historically in the integration code for updating context data fields https://github.com/segmentio/analytics.js-integrations/blob/6705f8c71eec21b88f91d8e6c891f7a47fe120e8/integrations/adobe-analytics/lib/index.js#L575

Reply:
I'm assuming the correct way to do this is to do `track.proxy('context')`?

Reply:
Hm. You're right it is previously used and when I try `msg.context()` in my node console it returns the correct object. I'm probably just reading too much into the docs and code.